fractions/microprofile/microprofile-health/src/main/resources/modules/org/eclipse/microprofile/health/main/module.xml

```diff
@@ -2,7 +2,8 @@
 <module xmlns="urn:jboss:module:1.3" name="org.eclipse.microprofile.health">
     <resources>
         <artifact name="org.eclipse.microprofile.health:microprofile-health-api:${version.microprofile-health}"/>
-        <artifact name="io.thorntail:health-api:${project.version}"/>
+        <artifact name="io.smallrye:smallrye-health:${version.smallrye.health}"/>
+        <artifact name="org.glassfish:javax.json:${version.org.glassfish.javax.json}"/>
```

Comment:
do you know what's the difference between using it as an artifact and as a module?
MP JWT depends on it this way: https://github.com/michalszynkiewicz/wildfly-swarm/blob/a61c6db0d897fd38a0f1f70e5c9e1be8f4844b32/fractions/microprofile/microprofile-jwt/module.conf#L15